app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/LoginViewModel.kt

```diff
@@ -1,83 +0,0 @@
-package org.greenstand.android.TreeTracker.viewmodels
```

Comment:
Let's keep all legacy files for now. Currently in master, if you switch the launch Activity to MainActivity the entire old version of the app should be 100% working.